### DISCUSSION TITLE
[Q] Looking for more maintainers?

### DISCUSSION THREAD

**Discussion by naorpeled** [exceljs/exceljs#2363] [Q&A]
Hey,
hope you're doing well.

Are you guys looking for more maintainers?
Would love to help maintain this awesome library :)

**Comment by sriharshAmur** [comment]
I'm willing to do the same as well. I've spent quite a bit of time with this package to help here.

**Comment by zurmokeeper** [comment]
I am also very happy to participate in this project, although I am not sure how much help can be provided, but to deal with some issues can still be, recently have been looking at the history of the issue, found that there are many are actually duplicate and can be closed, but suffer from the lack of permission can not operate.

**Comment by protobi-pieter** [comment]
Similar to #2194
